api/api.py

```diff
@@ -0,0 +1,188 @@
+from flask import Flask, request, stream_with_context, Response
+from flask_restful import inputs, reqparse, Resource, Api
+import json
+import jsonschema
+import io
+from PIL import Image
+import pkg_resources
+import threading
+import time
+
+import mcu.automation
+
+static_folder = pkg_resources.resource_filename(__name__, "/swagger")
+
+app = Flask(__name__, static_url_path="/", static_folder=static_folder)
```

Comment:
On my side, app does not work in browser by default. setting `static_url_path=""` solves the issue.
Can you try ? (don't forget to empty browser cache as resources may be cached)

Reply:
Super weird, as discussed offline I can't reproduce the issue (with the same flask version, 0.12.2). Anyway, I applied your patch, thanks.